### ISSUE TITLE
Pr documentation standardization and updating (Trac #647)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#778]
The Pr documentation does not use TeX for the equations so now looks a odd compared to rest of docs.

Also instructions need to be expanded to explain the fact that computations are constantly going on behind the scenes to recommend the optimum number of terms and the best reg constant to use.  The use thus only has to click on them to accept them.  Only if they really understand what they are doing should they actually enter a different value.  Also method assumes background is subtracted. If not they should use "estimate background."  Finally need to explain the explore dmax option.

Migrated from http://trac.sasview.org/ticket/647
```json
{
    "status": "closed",
    "changetime": "2016-09-06T06:04:44",
    "_ts": "2016-09-06 06:04:44.172035+00:00",
    "description": "The Pr documentation does not use TeX for the equations so now looks a odd compared to rest of docs.\n\nAlso instructions need to be expanded to explain the fact that computations are constantly going on behind the scenes to recommend the optimum number of terms and the best reg constant to use.  The use thus only has to click on them to accept them.  Only if they really understand what they are doing should they actually enter a different value.  Also method assumes background is subtracted. If not they should use \"estimate background.\"  Finally need to explain the explore dmax option.",
    "reporter": "butler",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-09-05T14:18:51",
    "component": "SasView",
    "summary": "Pr documentation standardization and updating",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView 4.0.0",
    "owner": "butler",
    "type": "task"
}
```

**Comment by butlerpd** [comment]
Trac update at `2016/09/05 15:17:32`: 
* **butler** changed owner from "" to "butler"
* **butler** changed status from "new" to "accepted"

**Comment by butlerpd** [comment]
Trac update at `2016/09/06 06:04:44`: 
* **butler** commented:

forgot to use fixes ticket when making the commit 0391dae728cc7e22bcc7ec329200722cd14f829f
Now closing manually.
      


* **butler** changed resolution from "" to "fixed"
* **butler** changed status from "accepted" to "closed"
